.github/workflows/autoclose.yaml

```
@@ -4,6 +4,7 @@ permissions:
   pull-requests: write
 
 on:
+  pull_request:
```

This should let a core dev reopen an auto-closed PR if need be:

GitHub's documentation [states](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request):

>  By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.

I haven't tested it, but it seems by default we'd have an auto-close loop.

```suggestion
  pull_request:
    types:
      - opened
```